tensorflow/core/ops/nn_ops.cc

```diff
@@ -1062,12 +1062,21 @@ REGISTER_OP("SoftmaxCrossEntropyWithLogits")
     .Attr("T: {half, bfloat16, float, double}")
     .SetShapeFn([](InferenceContext* c) {
       ShapeHandle input;
-      TF_RETURN_IF_ERROR(c->WithRank(c->input(0), 2, &input));
-      TF_RETURN_IF_ERROR(c->Merge(input, c->input(1), &input));
+      if (c->WithRank(c->input(0), 2, &input) == Status::OK() && c->Merge(input, c->input(1), &input) == Status::OK()) {
```

Comment:
Please keep lines below 80 characters. Use clangformat i in doubt.

Reply:
Thanks. Done.